app/scripts/controllers/reviewRegistration.js

```diff
@@ -121,14 +118,28 @@ angular
       });
     }
 
+    function getRegistrantType(typeId) {
+      return _.find(conference.registrantTypes, {
+        id: typeId,
+      });
+    }
+
+    function primaryRegType(currentRegistration) {
```

Comment:
Similar thing here too: 

```suggestion
    function primaryRegType(currentRegistration) {
      let primaryRegistrant = currentRegistration.registrants.find(
        registrant => registrant.id === currentRegistration.primaryRegistrantId,
      );
      return getRegistrantType(primaryRegistrant.registrantTypeId);
    }
```

Reply:
The same as for https://github.com/CruGlobal/conf-registration-web/pull/631/files#r324827925